x/protorev/keeper/keeper_test.go

```diff
@@ -57,6 +57,10 @@ func TestKeeperTestSuite(t *testing.T) {
 func (s *KeeperTestSuite) SetupTest() {
 	s.Setup()
 
+	poolManagerParams := s.App.PoolManagerKeeper.GetParams(s.Ctx)
+	poolManagerParams.TakerFeeParams.DefaultTakerFee = sdk.ZeroDec()
+	s.App.PoolManagerKeeper.SetParams(s.Ctx, poolManagerParams)
```

Comment:
Note to reviewer: This was set to zero here since I didn't understand the math being done to get the original number in the test. The result changed in a manner that felt right, but didnt want to just copy and paste the diff without understanding it.

Reply:
Follow-up note: this is also removed in a follow-up test PR that sets the taker fee default to 0